tests/test_imports.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2024 The HuggingFace Team. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+import subprocess
+
+from accelerate.test_utils.testing import TempDirTestCase, require_tuna
+from accelerate.utils import is_tuna_interpreter_available
+
+
+if is_tuna_interpreter_available():
+    from tuna._import_profile import read_import_profile
+    from tuna_interpreter import calculate_total_time
+    from tuna_interpreter.core import get_paths_above_threshold, sort_nodes_by_total_time
+
+
+def convert_list_to_string(data):
+    end_result = ""
+    arrow_right = "->"
+    for path in data:
+        end_result += f"{arrow_right.join(path[0])} {path[1]:.3f}s\n"
+    return end_result
+
+
+def run_import_time(command: str):
+    output = subprocess.run(["python3", "-X", "importtime", "-c", command], capture_output=True, text=True)
+    return output.stderr
+
+
+@require_tuna
+class ImportSpeedTester(TempDirTestCase):
+    clear_on_setup = False
+
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        output = run_import_time("import torch")
+        with open(f"{cls.tmpdir}/pytorch_results.log", "w") as f:
```

Comment:
This probably won't run on Windows, right? Not sure if it really matters though.

Reply:
Probably not, no. I don't believe toma supports windows

Reply:
`cls.tmpdir` is a `Path` object, so will fix